### PR TITLE
✨Use formatters option for stackdriver level

### PR DESCRIPTION
**Pull request by vlasy**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -82,13 +91,21 @@ const defaultLogger = (options: CosmasOptions & { loggerName?: string } = {}): C
     const messageKey = 'message'; // best option for Google Stackdriver,
     const streams = initLoggerStreams(defaultLevel, Object.assign({}, options, { messageKey }));
 
-    options.ignoredHttpMethods = options.ignoredHttpMethods || ['OPTIONS'];
+    const formatters: { level?: (label: string, level: number) => object } = {};
```
Comment (grissius) [inline src/index.ts:94]: Maybe we can use `pino.LoggerOptions['formatters']` from pino?

File: src/interfaces.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface CosmasOptions {
     defaultLevel?: pino.LevelWithSilent;
     disableStackdriverFormat?: boolean;
     streams?: CosmasStream[];
+    formatters?: object;
```
Comment (grissius) [inline src/interfaces.ts:21]: Same goes here for the types

Comment (vlasy) [reply src/interfaces.ts]: Fixed in c772474e33fb04625ca748dbbd747474842e223f

File: src/index.ts (hunk 1 above)
Comment (vlasy) [reply src/index.ts]: Fixed in c772474e33fb04625ca748dbbd747474842e223f

**Pull request merged**